rfcs/0013-avalanche.md

```diff
@@ -0,0 +1,98 @@
+# Replication via Avalanche 
+
+This RFC describes the current single layer replication mechanism as well as proposed changes to add a second layer and implementing an Avalanche mechanism.
+
+## Current Design
+There's two basic parts to the current Replication design. 
+
+#### Broadcast stage:
+In this stage, the leader distributes its data across the Layer-1 nodes. Currently, Layer-1 nodes are all known "tvu peers" (ClusterInfo::tvu_peers). 
+That way each Layer-1 node only receives partial data from the leader and the Replicate Stage in each Layer-1 node's TVU will ensure all data is shared between its Layer-1 peers 
```

Comment:
retransmit stage...  replicate stage is the "run it through the bank" stage

Reply:
whoops, yeah my bad.

Reply:
ReplicateStage is a bad name by the way, since we have replicator clients now. Consider sending a separate PR that renames it to something like ReplayStage so that we never need to see ReplicateStage in an RFC. :)